### ISSUE TITLE
Usage of deprecated Service-Component

### ISSUE THREAD

**Issue by jest** [osgi/v2archive.enroute.workspace#1]
`Service-Component` has been deprecated in bnd 3.5 and ATM the workspace based on this template fails to compile:

```
Service-Component entry '*' is not a FQN. Bnd no longer supports the deprecated Bnd DS annotations.
```

Probably should be removed from `cnf/ext/enroute.bnd`?

**Comment by bjhargrave** [comment]
The issue would have been with the latest Bnd 4.0 dev builds which have removed support for the Bnd DS annotations.

**Comment by jest** [comment]
Right, my ATM == master, sorry. :)

**Comment by simonjhy** [comment]
how to upgrade this in bnd.bnd?

**Comment by bjhargrave** [comment]
> how to upgrade this in bnd.bnd?

See the fix in 3f68bafd1b10d2c7ff1b39e360b2da043bc9f345
